config.analysis

```diff
@@ -131,6 +134,18 @@ compare_with_obs = True
 # output, N_movavg=12 corresponds to a 12-month moving average window)
 N_movavg = 12
```

Comment:
I would remove the moc_timeseries section, and add everything that is in there under moc_postprocess. This is because we will want to add the MOC time series to the current script that does the MOC vertical section.